.ci/jenkins_job_build.sh

```diff
@@ -169,6 +169,17 @@ popd
 
 "${ci_dir_name}/setup.sh"
 
+# Run unit tests on non x86_64
+if [ "$arch" != "x86_64" ]; then
+        echo "Running unit tests"
```

Comment:
@Amulyam24 yes, of course. Rust is installed to `~/.cargo/bin` via Rustup and even if the addition of that to `$PATH` is in the shell profile, the shell profile is never run between its installation and this run of the unit tests. I think the shortest, future-proof way to fix this would be something like

```suggestion
        echo "Running unit tests"
        cargo_env=~/.cargo/env
        [ -e "${cargo_env}" ] || "${ci_dir_name}/install_rust.sh" && source "${cargo_env}"
```

Reply:
Yes, makes sense. Thanks a lot @Jakob-Naucke ! Will update the PR.